TheFourFifths/consus
# Server-side store

### Summary

This PR updates the stores on the server to handle item check-in requests.
### Checklist

Finally, is it ready to be reviewed and merged: yes :cactus: 
### How to Review
1. Verify that checked-in items are removed from the student's item list.
2. Verify that checked-in items are available for check-out.
### Links
- [TFF-74](https://msoese.atlassian.net/browse/TFF-74)
